day1OfJS/variable.js

```diff
@@ -0,0 +1,65 @@
+/*
+Create a variable.js file and declare variables and assign string, boolean, undefined and null data types
+
+let vr = "javascript";
+
+let tr = true;
+let fl = false; 
```

Comment:
The name of the variables should be more telling

Reply:
Changed varibales - ea48586